packages/clay-charts/src/DonutChart.soy

```diff
@@ -4,5 +4,7 @@
  * This renders the component's whole content.
  */
 {template .render}
-	<div class="metal-chart"></div>
+	{@param? loading: bool}
+
+	{call Chart.chart data="all" /}
```

Comment:
@julien we should avoid the use of `data="all"` and try to explicitly pass the params.

Reply:
https://hosting-liferayfrontendguidelines.wedeploy.io/soy/conventions.html